README.md

```diff
@@ -11,6 +11,12 @@ the nightly release of the Rust compiler.
 
 This readme is for developing docs.rs. See [the about page](https://docs.rs/about) for user-facing documentation.
 
+
+## How has the rust doucumentation been rendered
+
+rustdoc generates documentation for Rust project, its respository in [here](https://github.com/rust-lang/rust/tree/master/src/librustdoc).
```

Comment:
docs.rs uses [rustdoc](https://github.com/rust-lang/rust/tree/master/src/librustdoc) to generate the documentation for every crate release on crates.io.

Reply:
I had adopt your advice already.